src/utilities.py

```diff
@@ -387,12 +387,10 @@ def complete_role(var, wrapper, role):
         else:
             matches = complete_match(role, var.ROLE_GUIDE.keys() | special_keys)
         if not matches:
-            wrapper.reply(messages["no_such_role"].format(role))
-            return False
+            return []
         if len(matches) > 1:
-            wrapper.reply(messages["ambiguous_role"].format(", ".join(matches)))
-            return False
-        return matches[0]
-    return role
+            return matches
+        return matches
```

Comment:
This whole block can simply be replaced by `return matches` right after the `complete_match` call.